limitador-server/src/envoy_rls/server.rs

```diff
@@ -246,7 +246,7 @@ mod tests {
             vec!["app_id"],
         );
 
-        let limiter = RateLimiter::default();
+        let limiter = RateLimiter::new(10_000);
```

Comment:
I'm missing how this work, but I don't understand why is this change needed? Shouldn't invoke the `default` set the expected cache_size?